tools/topology/topology2/production/CMakeLists.txt

```
@@ -22,6 +25,6 @@ foreach(tplg ${TPLGS})
 	  "${CMAKE_CURRENT_SOURCE_DIR}/../${input}" "${output}"
 	  "${CMAKE_CURRENT_SOURCE_DIR}/../" "${defines}")
 
-	add_custom_target(topology2_avs_${output} DEPENDS ${output}.tplg)
-	add_dependencies(topology2_cavs topology2_avs_${output})
+	add_custom_target(topology2_${output} DEPENDS ${output}.tplg)
+	add_dependencies(topology2_prod topology2_${output})
```

Nit
```suggestion
	add_dependencies(topology2_prod topology2_prod_${output})
```
See the difference in `ninja -C tools/build_tools/ help`

(I know, should have said earlier)